_includes/navigation.html

```diff
@@ -1,11 +1,18 @@
 {% assign navurl = page.url | remove: 'index.html' %}
-<ul>
-{% for item in include.navigation %}
-  <li>
-    <a href="{{ item.url }}"{% if item.url == navurl %} data-current="current page"{% endif %}>{{ item.text }}</a>
-    {% if item.subitems %}
-      {% include navigation.html navigation=item.subitems %}
-    {% endif %}
-  </li>
-{% endfor %}
-</ul>
+<div class="nav-split--main">
```

Comment:
modifier classes are meant to be used with base classes, e.g. `nav-split--main`, but there's no `nav-split`, so that's one indication that this is not great.
if the parent was`nav` I'd probably call this `nav__main` and `nav__sub` (or `__aside` or `__whatever`)